css_portfolio.css

```diff
@@ -0,0 +1,146 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-size: 62.5;
+	margin: 0px;
+	font-family: helvetica;
+}
+
+.navbar {
+	display: flex;
+	justify-content: space-between;
+	height: 55px;
+	background-color: rgb(30, 30, 30);
+	width: 100%;
+	position: fixed;
+	top: 0;
+	padding-left: 30px;
+	padding-right: 30px;
+}
+
+@media only screen and (max-width: 700px) {
+	.navbar {
+		padding-left: 5px;
+		padding-right: 5px;
+	}
+}
+
+.pagelinks {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	width: 500px;
+	list-style-type: none;
+	font-size: 1.3rem;
+	padding: 0px;
+	margin-left: 25px;
+	margin-top: 15px;
+}
+
+.pagelinks a:link {
+	text-decoration: none;
+	color: rgb(255, 255, 255);
+}
+
+.pagelinks a:visited {
+	text-decoration: none;
+	color: rgb(255, 255, 255);
+}
+
+@media only screen and (max-width: 700px) {
+	.pagelinks {
+		width: 80px;
+	}
+}
+
+.nav {
+	border-color: rgb(255, 255, 255);
+	border-width: 2px;
+	border-style: solid;
+	border-radius: 25px;
+	padding: 7px;
+}
+
+@media only screen and (max-width: 700px) {
+	.nav {
+		display: none;
+	}
+}
+
+.menu {
+	visibility: hidden;
+	border-color: rgb(255, 255, 255);
+	border-width: 2px;
+	border-style: solid;
+	border-radius: 25px;
+	padding: 7px;
+}
+
+@media only screen and (max-width: 700px) {
+	.menu {
+		visibility: visible;
+	}
+}
+
+.cv {
+	font-size: 1.5rem;
+	margin-top: 10px;
+	border-color: rgb(255, 255, 255);
+	border-width: 2px;
+	border-style: solid;
+	border-radius: 25px;
+	width: 50px;
+	height: 35px;
+	padding-top: 2px;
```

Comment:
As this div is in a flex container would justify-content: center; achieve the same effect with cleaner code?